### PR TITLE
Add test command step fileserver

### PR DESCRIPTION
**Pull request by maraino**
### Description
This PR adds a new hidden command `step fileserver`. 

This command starts a file server on a given directory, it supports HTTP and HTTPS with a given certificate file.

### CODE REVIEW

File: command/fileserver/fileserver.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+)
+
+func init() {
+	cmd := cli.Command{
+		Name:   "fileserver",
+		Hidden: true,
+		Action: command.ActionFunc(fileServerAction),
+		Usage:  "start an HTTP(S) server serving the contents of a path",
+		UsageText: `step fileserver <root>
+		[--address=<address>] [--cert=<path>] [--key=<path>`,
```
Comment (dopey) [inline command/fileserver/fileserver.go]: missing close bracket after key

File: command/fileserver/fileserver.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	"github.com/smallstep/cli/command"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	cmd := cli.Command{
+		Name:   "fileserver",
+		Hidden: true,
+		Action: command.ActionFunc(fileServerAction),
+		Usage:  "start an HTTP(S) server serving the contents of a path",
```
Comment (dopey) [inline command/fileserver/fileserver.go]: maybe say "contents of a directory" since you're calling the arg <dir>

Comment (maraino) [reply command/fileserver/fileserver.go]: 👍

Comment (maraino) [reply command/fileserver/fileserver.go]: the usage would be too long to keep it in one line, leaving it as path

**Pull request merged**